config/ember-try.js

```diff
@@ -1,39 +1,52 @@
 module.exports = {
   scenarios: [
     {
-      name: 'Ember Canary',
+      name: 'default',
+      dependencies: { }
+    },
+    {
+      name: '1.10.1',
       dependencies: {
-        "ember": "components/ember#canary"
-      },
-      resolutions: {
-        "ember": "canary"
+        'ember': '1.10.1'
       }
     },
     {
-      name: 'Ember Beta',
+      name: '1.11.3',
       dependencies: {
-        "ember": "components/ember#beta"
-      },
-      resolutions: {
-        "ember": "beta"
+        'ember': '1.11.3'
```

Comment:
What are your thoughts on using tilde dependencies for testing previous release versions?